### PR TITLE
cli: add --human flag to repo stat & stats repo

### PR DESCRIPTION
**Pull request by chirag-shinde**
- Added a `humanize` object to /src/cli/utils with `Bytes` method
that takes bytes and returns Human readable size.
- Used `humanize.Bytes()` method in /commands/stats/repo and
/commands/repo/stat if `--human` flag is passed.

solves #1996 

License: MIT
Signed-off-by: Chirag Shinde <shinde.chirag2204@gmail.com>

### CODE REVIEW

**Comment by chirag-shinde** [comment]
Screenshot of the output
Screenshot 2019-05-23 at 8 56 38 PM

File: src/cli/utils.js [hunk 1]
```diff
@@ -135,3 +135,18 @@ exports.singleton = create => {
...
+    }
+    const units = ['kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+    let unit = -1
+    do {
+      bytes /= thresh
+      ++unit
+    } while (Math.abs(bytes) >= thresh && unit < units.length - 1)
+    return bytes.toFixed(1) + ' ' + units[unit]
+  }
+}
```
Comment (alanshaw) [inline src/cli/utils.js]: Can we please use a tested module on npm for this? `ipfs-http-response` already uses `filesize` - can we use the same module?

**Comment by chirag-shinde** [comment]
I'd like some help with writing the unit test for handler please. 
As it is returning a string of stats I'm not sure how to go about testing for the `--human` flag as the only difference between the two strings would be with or without size units.
Thanks.

**Comment by daviddias** [comment]
Hi! js-ipfs master just got a whole new set of automated tests with #2528, https://github.com/ipfs/js-ipfs/pull/2440 and also running some of the test suites from our early testers (hi5 to  @achingbrain for setting it all up!). **Would you mind rebasing the master branch on this PR to ensure it runs all the latest tests? Thank you!**

**Pull request closed without merging**